build_tools/docker/nvidia/Dockerfile

```
@@ -50,3 +50,10 @@ COPY --from=fetch-nvidia \
 RUN apt-get install "/tmp/${NVIDIA_COMMON_DEB?}" \
   "/tmp/${NVIDIA_GL_DEB?}" \
   "/tmp/${NVIDIA_COMPUTE_DEB?}"
+
+# install cuda sdk
+RUN wget https://developer.download.nvidia.com/compute/cuda/11.6.0/local_installers/cuda-repo-debian11-11-6-local_11.6.0-510.39.01-1_amd64.deb
+RUN dpkg -i cuda-repo-debian11-11-6-local_11.6.0-510.39.01-1_amd64.deb
+RUN apt-key add /var/cuda-repo-debian11-11-6-local/7fa2af80.pub
+RUN apt-get update
+RUN apt-get -y install cuda-toolkit-11-6
```

I thought I commented on this before, but can't find it. Each `RUN` statement creates a new layer in the docker image, bloating the image. You also always want to have `apt-get update` as part of the same `RUN` statement as `apt-get install` (see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get)

```suggestion
RUN wget https://developer.download.nvidia.com/compute/cuda/11.6.0/local_installers/cuda-repo-debian11-11-6-local_11.6.0-510.39.01-1_amd64.deb \
  && dpkg --install cuda-repo-debian11-11-6-local_11.6.0-510.39.01-1_amd64.deb \
  && apt-key add /var/cuda-repo-debian11-11-6-local/7fa2af80.pub \
  && apt-get update \
  && apt-get -y install cuda-toolkit-11-6
```